crates/base/src/server.rs

```diff
@@ -87,24 +90,43 @@ impl Server {
         ip: &str,
         port: u16,
         main_service_path: String,
+        maybe_events_service_path: Option<String>,
```

Comment:
Maybe ?

Reply:
since it's an option type maybe isn't really necessary.